data_steward/validation/main.py

```diff
@@ -824,9 +904,19 @@ def _get_submission_folder(bucket, bucket_items, force_process=False):
             item for item in bucket_items
             if item['name'].startswith(folder_name)
         ]
+
+        # filter files based on age
         submitted_bucket_items = list_submitted_bucket_items(
             folder_bucket_items)
 
+        # determine if the above is missing any of the "required" files
+        missing_required = _get_missing_required_files(submitted_bucket_items)
+        if len(missing_required) > 0:
```

Comment:
My thought process is the same here.  I would suggest:
```suggestion
    if missing_required:
```
It seems clearer, especially with the well named variable.